### ISSUE TITLE
WCAG: Input fields for activation codes (when logged in)

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2239]
These input fields are made from non-interactive div elements. For the auxiliary software it is not clear that these are input fields and which data must be entered here. See screenshot 9.

The reading software cannot tell its user that these are buttons because they miss the correct role. Now these buttons are coded as neutral (non-interactive) div elements. In addition, the reading software does not know what the function (name) of these buttons is.

The most robust solution is to use the native HTML elements such as input elements. Another possible solution is to use WAI-ARIA. 

![image.png](https://images.zenhubusercontent.com/5ba4ae539f1e9e096ebdcc88/2ea2a513-0817-418e-b0b2-056967424fe2)
